src/main/drivers/accgyro/accgyro_spi_bmi160.c

```diff
@@ -98,14 +98,14 @@ uint8_t bmi160Detect(const busDevice_t *bus)
         return BMI_160_SPI;
     }
 
-    spiSetDivisor(bus->busdev_u.spi.instance, spiCalculateDivider(BMI160_MAX_SPI_CLK_HZ));
+    spiSetClkDivisor(dev, spiCalculateDivider(BMI160_MAX_SPI_CLK_HZ));
 
     /* Read this address to activate SPI (see p. 84) */
-    spiBusReadRegister(bus, 0x7F);
+    spiReadRegMsk(bus, 0x7F);
```

Comment:
SPI API explanatory comments added in `bus_spi.h`, notably:

```
/*
 * Routine naming convention is:
 * 	spi[Read][Write][Reg][Msk][Buf][RB]
 *
 * 		Read: Perform a read, returning the value read unless 'Buf' is specified
 * 		Write Perform a write
 * 		ReadWrite: Perform both a read and write, returning the value read unless 'Buf' is specified
 * 		Reg: Register number 'reg' is written prior to the read being performed
 * 		Msk: Register number is logically ORed with 0x80 as some devices indicate a read by accessing a register with bit 7 set
 * 		Buf: Pass data of given length by reference
 * 		RB:  Return false immediately if the bus is busy, otherwise complete the access and return true
 */
```